contributor_environment.yml

```diff
@@ -15,3 +15,8 @@ dependencies:
   - black
   - isort
   - interrogate
+  - coverage
+
+
+=======
```

Comment:
looks like this might be an artifact from a merge conflict resolution?

Reply:
I thought that someone else put that in. I will remove it.